napari/layers/intensity_mixin.py

```diff
@@ -18,7 +18,7 @@ class IntensityVisualizationMixin:
 
     When used, this should come before the Layer in the inheritance, e.g.:
 
-        class Image(ImageSurfaceMixin, Layer):
+        class Image(IntensityVisualizationMixin, Layer):
```

Comment:
Another good catch!

Nit: `Image` is subclass of `Layer` that we define, but its declaration is `class Image(_ImageBase)` where we have `class _ImageBase(IntensityVisualizationMixIn, Layer)`, so this might be confusing. But I guess this is just an example, so this `Image` does not have to correspond to our `Image`, so you can also ignore me here.

Reply:
Right, happy to change it to `_ImageBase` if that is preferred

Reply:
Well, `_ImageBase` is protected, so maybe it's actually worse. You could consider `Surface` as that (at least currently) does immediately inherit `IntensityVisualizationMixin`. Or you can just leave this as is. I'll leave it up to you.